Exercise1.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+
+def is_permutation(string1, string2) :
+	# check if any of the strings is None
+	if string1 is None or string2 is None:
+		return False
+	# change strings inputted to dictionaries
+	string1_dict = string_to_dict(string1)
+	string2_dict = string_to_dict(string2)
+	
+	# check if the two dictionaries are equal
+	result = dicts_are_equal(string1_dict, string2_dict)
+	
+	return result
+	
+def string_to_dict(input_string) :
+	# make sure all characters are lower case
+	lower_case_string = input_string.lower()
+	
+	# create a dictionary where you map each character with a counter of
+	# how many instances of a character exist
+	# if the strings can only include characters from the English alphabet:
+	# initialise the dictionary at this stage with all the letters of the alphabet
+	# and a starting counter value of 0
+	dict = {}
+	
+	# add the characters to the dictionary
+	for character in lower_case_string:
+		if character in dict:
```

Comment:
I am not really sure how to use get in lines 29/30 but I did use it in line 31 to avoid saving the counter in a value variable first.